src/pages/auth/index.tsx

```diff
@@ -17,11 +28,21 @@ export default function Auth() {
     }));
   };
 
+  const handleSubmit = (event: { preventDefault: () => void }) => {
+    event.preventDefault();
+    axios.post(`${process.env.NEXT_PUBLIC_API_URL}/login`, inputValue).then((response) => {
```

Comment:
this should be inside a try catch to ... ya know, catch the backend error if it happens

Reply:
when we use react query, the library will take care of that for us, but here we need to handle it ourselves

Reply:
also, on that note, please add a loading state so we don't send a million requests if the user super click-happy ... and also to show a nice spinner

Reply:
might also want to show an error? like what happens if the endpoint fails because there's no internet vs what happens if the enpoint fails because of bad credentials